integration-test/src/main/resources/testsuites/e2e/aws-e2e-tests.yaml

```diff
@@ -16,3 +16,4 @@ tests:
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRecipeTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRangerRazEnabledTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.InternalSdxRepairWithRecipeTest
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.rotatesaltpassword.RotateSaltPasswordE2ETest
```

Comment:
Is AWS the only affected provider here or validations are enough only on AWS to be rest assured this is working?

Reply:
It is not provider specific feature, so it is enough to test it on a single provider

Reply:
```
...
Caused by: org.testng.TestNGException: 
Cannot find class in classpath: com.sequenceiq.it.cloudbreak.testcase.e2e.rotatesaltpassword.RotateSaltPasswordE2ETest
	at org.testng.xml.XmlClass.loadClass(XmlClass.java:75)
...
```
The path here is not valid anymore